### DISCUSSION TITLE
VCHI not initialized

### DISCUSSION THREAD

**Discussion by rottenoasis** [mcguirepr89/BirdNET-Pi#661] [General]
I'm getting this error in the log "VCHI initialization failed". I tried this command after doing some googling:
 sudo usermod -aG video [username]
And still same result/no change. And ideas on how to resolve this? I don't see any adverse effects, I just don't like seeing errors, lol!

**Comment by jmtuts** [comment]
I have the same error:  Tools -> System Info -> At the top left I see a red triangle with an exclamation mark.  When I click this I see the following:

![image](https://user-images.githubusercontent.com/123665555/221041026-5e79a748-ebde-4978-9fa1-5cb0c47747e6.png)

**Reply by ehpersonal38** [reply]
Did you try the command the OP mentioned?
